ext4: remove redundant code in ext4_fill_super()

From: Li Zefan <lizf@cn.fujitsu.com>

The previous sb_min_blocksize() has already set the block size.

This piece of code was introduced in commit:
	afc7cbca5bfd556c3e12d3acefbee5ab0cbd4670
	(ext4: Support large blocksize up to PAGESIZE)
And it should be a mistake.

Signed-off-by: Li Zefan <lizf@cn.fujitsu.com>
Signed-off-by: Mingming Cao <cmm@us.ibm.com>
---
 fs/ext4/super.c |    5 -----
 1 file changed, 5 deletions(-)

Index: linux-2.6.26-rc5/fs/ext4/super.c
===================================================================
--- linux-2.6.26-rc5.orig/fs/ext4/super.c	2008-06-06 17:03:47.000000000 -0700
+++ linux-2.6.26-rc5/fs/ext4/super.c	2008-06-06 17:03:48.000000000 -0700
@@ -1899,11 +1899,6 @@ static int ext4_fill_super (struct super
 		goto out_fail;
 	}
 
-	if (!sb_set_blocksize(sb, blocksize)) {
-		printk(KERN_ERR "EXT4-fs: bad blocksize %d.\n", blocksize);
-		goto out_fail;
-	}
-
 	/*
 	 * The ext4 superblock will not be buffer aligned for other than 1kB
 	 * block sizes.  We need to calculate the offset from buffer start.
